.github/workflows/CI.yml

```diff
@@ -44,3 +53,13 @@ jobs:
     - name: Publish nuget package to nuget.org
       run: dotnet nuget push nugetPackage\*.nupkg --api-key "${{ secrets.GHPackagesToken }}" --source "https://www.nuget.org/"
       shell: pwsh
+    - name: Create Release
```

Comment:
add a condition (or the actions equivalent) to only create a new release on PR builds

Reply:
you mean, only create a new release on the main trunk? I wouldn't want a PR release, only a release off Main. Following?